packages/api/lib/writeRecords/write-granules.js

```diff
@@ -317,12 +409,12 @@ const _writeGranule = async ({
   );
   log.info('Successfully wrote granule %j to DynamoDB', dynamoGranuleRecord);
 
-  const { files, granuleId, status, error } = dynamoGranuleRecord;
+  const { files, status, error } = dynamoGranuleRecord;
 
   await _writeGranuleFiles({
     files,
     granuleCumulusId,
-    granuleId,
+    granule: dynamoGranuleRecord,
```

Comment:
If you're going to do this, which is fine, then you can remove this above

`const { files, status, error } = dynamoGranuleRecord;`

and just destructure those properties that inside `_writeGranuleFiles({`

Reply:
good catch. done.